prdoc/pr_7866.prdoc

```diff
@@ -0,0 +1,12 @@
+title: Make litep2p the default backend in Kusama
+
+doc:
+  - audience: [Node Dev, Node Operator]
+    description: |
+      This PR makes the litep2p backend the default network backend in Kusama.
```

Comment:
nit: Would that be worth mentioning that `kusama|ksm` prefix is being checked in spec `id`?

Reply:
Managed to get this bit of info from polkadot/service, thanks 🙏